broScripts/parse_pcap.py

```diff
@@ -82,8 +107,7 @@ def is_person_empty(field):
                     continue
                 if line[0] == '[' and line[-2] == ']': # Check that this line represents an email
                     try:
-                        line = line.replace("\\'", "\'")
-                        headers = eval(line)
+                        headers = parseLine(line)
```

Comment:
No, we do not have to modify Mike'e Inbox class because this parser returns an output consistent with what the Inbox class expects.